library_conversion/tools/library_conversion/nist_msp_conversion.py

```diff
@@ -1,26 +1,24 @@
 #!/usr/bin/python
```

Comment:
This file should not be included

Reply:
I added a test file for other msp formats that are compatible with the NIST specs.
This format is also used by MZmine and other tools to create libraries.